declarations/panorama/dota_panels.d.ts

```diff
@@ -130,84 +137,62 @@ interface Panel {
     SetAttributeString(name: string, value: string): void;
     SetAttributeUInt32(name: string, value: number): void;
 
-    SetInputNamespace(naespace: string): void; // ??
+    SetInputNamespace(namespace: string): void; // ??
 
     RegisterForReadyEvents(callback: (event: object) => void): void; // ????
 
     BReadyForDisplay(): boolean;
     SetReadyForDisplay(value: boolean): void; // ???
-    SetPanelEvent(event: PanelEvent, handler: () => void): void;
-
-    RunScriptInPanelContext(script: string): void;
 }
 
-interface VCSSStyleDeclaration extends CSSStyleDeclaration {
-    /** 
-     * Controls blending mode for the panel.  See CSS mix-blend-mode docs on web, except normal for us is with alpha blending.
-     * 
+interface VCSSStyleDeclaration {
```

Comment:
Why is the extension gone? Are its fields not nullable?

Reply:
To improve autocomplete results. All possible values are included.